contracts/utils/Ownable.sol

```diff
@@ -94,4 +93,20 @@ abstract contract Ownable is IOwnable {
             sstore(_OWNER_SLOT, newOwner)
         }
     }
+
+    /**
+     * @notice Internal function to transfer ownership of the contract to a new account `newOwner`.
+     * @dev Called in the constructor to set the initial owner.
+     * @param newOwner The address to transfer ownership to
+     */
+    function _transferOwnership(address newOwner) internal virtual {
+        if (newOwner == address(0)) revert ZeroAddress();
```

Comment:
```suggestion
        if (newOwner == address(0)) revert InvalidAddress();
```
Perhaps?

Reply:
Yeah also needs to be renamed in the interface

Reply:
Changed to `InvalidOwnerAddress `